Tests/Tests.Unity/Assembly-CSharp-Editor.csproj

```diff
@@ -314,42 +314,18 @@
     <Reference Include="Mono.Cecil">
       <HintPath>C:\work\dotnet\Realm\Realm.Unity\Editor\Cecil\Mono.Cecil.dll</HintPath>
     </Reference>
-    <Reference Include="Nito.AsyncEx.Tasks">
```

Comment:
Are these dependences gone because we only do the bundled version of the package for unity, right?